uprotocol/client/usubscription/v3/inmemoryusubcriptionclient.py

```diff
@@ -47,7 +55,7 @@ class MyNotificationListener(UListener):
     def __init__(self, handlers):
```

Comment:
Add a docsting to the class to keep it consistent everywhere

Reply:
Added